aspnetcore/fundamentals/app-state.md

```diff
@@ -62,6 +62,7 @@ Session state exhibits the following behaviors:
 * Session data is deleted either when the <xref:Microsoft.AspNetCore.Http.ISession.Clear%2A?displayProperty=nameWithType> implementation is called or when the session expires.
 * There's no default mechanism to inform app code that a client browser has been closed or when the session cookie is deleted or expired on the client.
 * Session state cookies aren't marked essential by default. Session state isn't functional unless tracking is permitted by the site visitor. For more information, see <xref:security/gdpr#tempdata-provider-and-session-state-cookies-arent-essential>.
+* Note: there is no replacement for the cookieless session feature from old ASP.NET Framework because it's considered insecure and can lead to session fixation attacks.
```

Comment:
```suggestion
* **Note**: There is no replacement for the cookieless session feature from the ASP.NET Framework because it's considered insecure and can lead to session fixation attacks.
```